device/mellanox/x86_64-mlnx_msn4800-r0/platform.json

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
besides interface info, there is some other info also need to be included in this file, please refer to 2700 https://github.com/Azure/sonic-buildimage/blob/master/device/mellanox/x86_64-mlnx_msn2700-r0/platform.json#L2

Reply:
I added the chassis info to platform.json.
I noticed that on the earlier platform version (3700, 4600, 4700...) the chassis info is missing also.